test/data/records.json

```diff
@@ -0,0 +1,6 @@
+[
+    {"lfn":"/store/file.root",
+     "pfn":"test/data/testdata.txt",
+     "block":"abc-123",
```

Comment:
block convention in CMS is the /a/b/c#123, i.e. first part is a dataset path followed by pound and block unique id. Please adjust it here.